ros/api/v0/hosts.py

```diff
@@ -184,12 +186,14 @@ def get(self, host_id):
 
         if profile:
             record = {'inventory_id': host_id}
-            record['display_performance_score'] = profile.display_performance_score
             record['rating'] = rating_record.rating if rating_record else None
-            record['recommendation_count'] = len(system.rule_hit_details)
-            record['state'] = system.rule_hit_details[0].get('key')
+            record['report_date'] = profile.report_date
```

Comment:
Could you please include rule hits details same as `/systems` API?

Reply:
We do not have `rule_hit_details` **shown** in /systems response. It is just used to calculate recommendation count currently. And once we start storing the rec_count and state in the coulmn we might not need to include it in /systems call either.

Reply:
On system detail page, there is recommendations list below as per mockup. 
Please let me know how backend is going to send those details. As per this, frontend will consume data.